utils/indigo-depict/main.c

```diff
@@ -901,7 +901,7 @@ int main(int argc, char* argv[])
         _prepare(obj, p.aromatization);
         if (p.action == ACTION_LAYOUT)
         {
-            indigoLayout(obj);
+            // indigoLayout(obj);

```

Comment:
why the layout is commented? it is in action ACTION_LAYOUT

Reply:
fixed